02-clients/references/client.yml

```diff
@@ -311,50 +301,17 @@ methods:
           default: N/A
     java:
       <<: *method-users-delete
-      method: client.users().delete(String username);
+      method: client.users().get(String user).delete();
```

Comment:
I do not find this in the java client.
The signature I see is the one that's been removed.

Reply:
Fixed by updating API reference to the latest master branch of public docs